firefox_ui_tests/remote/security/test_mixed_content.py

```diff
@@ -0,0 +1,40 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness.testcase import FirefoxTestCase
+
+
+class TestMixedContent(FirefoxTestCase):
+    def setUp(self):
+        FirefoxTestCase.setUp(self)
+
+        self.test_url = 'https://mozqa.com/data/firefox/security/mixedcontent.html'
+
+    def test_mixed_content(self):
+        with self.marionette.using_context('content'):
+            self.marionette.navigate(self.test_url)
+        favicon = self.browser.navbar.locationbar.favicon
+
+        def check_favicon_image(self):
+            favicon_image = self.execute_script("""
+                return arguments[0].ownerDocument.defaultView
+                                   .getComputedStyle(arguments[0])
+                                   .getPropertyValue('list-style-image');
```

Comment:
For js code we use double quotes. Also it would be necessary to have this code as a shared method given that it is used at 5 different places for security tests. We can have it under `firefox_puppeteer/api/dom.py`. Would you mind doing that? If not we need a follow-up bug and maybe @galgeek can work on it.